cosmo_tester/test_suites/snapshots/multi_tenant_snapshot_test.py

```diff
@@ -13,141 +13,182 @@
 #    * See the License for the specific language governing permissions and
 #    * limitations under the License.
 
-import json
-import os
-import uuid
-
 import pytest
 
-from cosmo_tester.framework.cluster import (
-    CloudifyCluster,
-    MANAGERS,
-)
-from cosmo_tester.framework.util import assert_snapshot_created
-
-from cloudify_cli.constants import DEFAULT_TENANT_NAME
-
 from . import (
-    deployment_id,
-    deploy_helloworld,
-    wait_for_execution,
-    update_client_tenant
+    assert_hello_worlds,
```

Comment:
I don't have anything concrete to say here, except that when I see a list this long, it makes me think that there's something wrong with the design. I would assume that only the high level functions would be necessary here, and the rest would be internal to the implementation.